manage-data/lifecycle/index-lifecycle-management/policy-apply.md

```diff
@@ -0,0 +1,72 @@
+---
+navigation_title: Apply a policy to an index
+applies_to:
+  stack: ga
+products:
+  - id: elasticsearch
+---
+
+# Apply a lifecycle policy to an index [apply-policy-manually]
+
+When you create new {{es}} index you can use an index template to apply the lifecycle policy by which the index will be managed. This process is described in [Configure a lifecycle policy](/manage-data/lifecycle/index-lifecycle-management/configure-lifecycle-policy.md).
+
+You can also manually apply a lifecycle policy to an existing index, as described here. If an index is currently managed by an ILM policy you must first remove that policy before applying a new one. Refer to [Switch to a different lifecycle policy](/manage-data/lifecycle/index-lifecycle-management/policy-updates.md#switch-lifecycle-policies) for details.
+
+You can do this procedure in {{kib}} or using the {{es}} API.
+
+::::{warning}
+Do not manually apply a policy that uses the rollover action to an index which has not yet rolled over. Otherwise, the policy may not be carried forward when the rollover action creates a new index.
+::::
+
+::::{tab-set}
+:group: kibana-api
+:::{tab-item} {{kib}}
+:sync: kibana
+
+To apply a lifecycle policy to an existing index:
+
+1. Go to **Stack Management > Index Management**. In the **Indices** tab, search for and select the index that you created, for example `test-00001`. Enable **Include hidden indices** to see the full list of indices.
+
+1. From the **Manage index** dropdown menu select **Add lifecycle policy**.
+
+1. Choose a lifecycle policy and confirm your changes.
+
+Once the policy is applied, {{ilm-init}} starts managing the index immediately.
+:::
+
+:::{tab-item} API
+:sync: api
+Use the [update settings API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-put-settings) to apply a lifecycle policy to an index.
```

Comment:
Question on what you're going for vs what I'd expect this section to append: 

when you apply an ILM policy it _always_ sequentially runs from the top (so `hot` maybe `rollover`) so _even if_ your index previously rolled over or something and you apply a new ilm policy or one for the first time, you're going back to the top of the queue. so if you may actually want the `rollover_alias` and/or `indexing_complete` mentioned before and/or do an ILM Move step to move to wherever in the new policy flow you want to be.